nixos/platform/agent.nix

```diff
@@ -84,6 +84,14 @@ in
   };
 
   config = mkMerge [
+    {
+      # Write NixOS warnings to a file, separated by two newlines.
+      # We use that for `fc-manage check` to display (deprecation) warnings.
+      environment.etc."fcio_nixos_warnings".text =
```

Comment:
PR descritpion says `/etc/fcio/nixos/warnings", this says `/etc/fcio_nixos_warnings`.
If we go with the subdirectory structure, it might make sense at some point to unify the directory structure of all fc tools.